pkg/http/rest/status.go

```diff
@@ -0,0 +1,43 @@
+package rest
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+func HasValidContent(resp *http.Response) bool {
+	if resp != nil {
+		code := resp.StatusCode
+		return codeOK(code) && Is2xx(code) &&
+			code != http.StatusNoContent &&
+			resp.Body != nil
+	}
+
+	return false
+}
+
+func Is4xx(code int) (bool, error) {
+	if code >= http.StatusBadRequest && code <= 499 {
+		return true, errors.New(fmt.Sprintf("Received HTTP%d", code))
+	}
+
+	return false, nil
+}
+
+func Is5xx(code int) (bool, error) {
+	if code >= http.StatusInternalServerError && code <= 599 {
+		return true, errors.New(fmt.Sprintf("Received HTTP%d", code))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)